src/app/content/components/Page/contentDOMTransformations.ts

```diff
@@ -209,3 +210,18 @@ function moveFootnotes(document: Document, rootEl: HTMLElement, intl: IntlShape)
     sup.appendChild(link);
   }
 }
+
+const setLinksAttributes = (rootEl: HTMLElement) => {
+  for (const a of Array.from(rootEl.querySelectorAll('a'))) {
+    const href = a.getAttribute('href') || '';
+    if (href.startsWith('https://') || href.startsWith('http://') || href.startsWith('//')) {
+      // target blank and add `rel` to links that begin with: http:// https:// //
+      a.setAttribute('target', '_blank');
+      a.setAttribute('rel', 'noopener nofollow');
+    } else if (href.startsWith('../')) {
```

Comment:
relative content won't always start with this, eg page urls just have the page slug in the href

Reply:
oh but this is actually correct, the comment is just misleading.

the actual goal is to `// target blank and allow indexing links to other books`

Reply:
the desire is for links outside the book to be `target: blank` but we would be fine to allow indexing to all relative links